README.md

```diff
@@ -581,7 +581,7 @@ App Links Measurement Events sends additional information from App Links Intents
  [maven-tasks-link]: https://maven-badges.herokuapp.com/maven-central/com.parse.bolts/bolts-tasks
  [maven-applinks-svg]: https://img.shields.io/maven-central/v/com.parse.bolts/bolts-applinks.svg?label=bolts-applinks&style=flat
  [maven-applinks-link]: https://maven-badges.herokuapp.com/maven-central/com.parse.bolts/bolts-applinks
- [license-svg]: https://img.shields.io/badge/license-BSD-lightgrey.svg?style=flat
+ [license-svg]: https://img.shields.io/badge/license-MIT-lightgrey.svg?style=flat
  [license-link]: https://github.com/BoltsFramework/Bolts-Android/blob/master/LICENSE
 
  [latest]: https://search.maven.org/remote_content?g=com.parse.bolts&a=bolts-tasks&v=LATEST
```

Comment:
Can you add this to the very end of the README.md

```
## License
Bolts-Android is MIT licensed, as found in the LICENSE file.
```

Reply:
Done :)